src/Server/RestRouterHandlers/TableRestRouterHandler.cpp

```diff
@@ -312,7 +321,7 @@ String TableRestRouterHandler::getCreationSQL(const Poco::JSON::Object::Ptr & pa
 {
     const auto & time_col = getStringValueFrom(payload, "_time_column", "_time");
     std::vector<String> create_segments;
-    create_segments.push_back("CREATE TABLE " + database + "." + payload->get("name").toString());
+    create_segments.push_back("CREATE TABLE IF NOT EXISTS " + database + "." + payload->get("name").toString());
```

Comment:
We can't add `IF NOT EXISTS`, if table exists, we shall error out

Reply:
fixed